app/controllers/Coordinate.scala

```diff
@@ -9,7 +9,7 @@ final class Coordinate(env: Env) extends LilaController(env) {
       ctx.userId ?? { userId =>
         env.coordinate.api getScore userId map (_.some)
       } map { score =>
-        views.html.coordinate.home(score)
+        views.html.coordinate.coordinate.home(score)
```

Comment:
You can avoid the double "coordinate" by having it in a `home` file/object and using an `apply` method. See for example https://github.com/lichess-org/lila/blob/master/app/views/team/show.scala.

Reply:
Ah, that make sense. Thanks - fixed in 7c70b11